src/http/Application.php

```
@@ -168,6 +173,40 @@ public function coreComponents(): array
+     * while worker cleanup still runs.
+     *
+     * Usage example:
+     * ```php
+     * $psr7Response = $app->handle($psr7Request);
+     * (new \yii2\extensions\psrbridge\emitter\SapiEmitter())->emit($psr7Response);
+     * $app->finalize();
+     * ```
+     *
+     * @param bool $success Whether the runtime emitted the response successfully.
+     */
+    public function finalize(bool $success = true): void
```

PHPMD (2.15.0)

[error] The method finalize has a boolean flag argument $success, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)